core/controllers/cron.py

```diff
@@ -223,3 +227,55 @@ def get(self):
             cron_services.JobModelsCleanupManager.enqueue(
                 cron_services.JobModelsCleanupManager.create_new())
             logging.warning('Deletion jobs for JobModels entities kicked off.')
+
+
+class CronMailReviewersContributorDashboardSuggestionsHandler(
+        base.BaseHandler):
+    """Handler for mailing reviewers suggestions on the Contributor
+    Dashboard that need review.
+    """
+
+    @acl_decorators.can_perform_cron_tasks
+    def get(self):
+        """Sends each reviewer an email with up to
+        suggestion_services.MAX_NUMBER_OF_SUGGESTIONS_TO_EMAIL_REVIEWER
+        suggestions that have been waiting the longest for review, based on
+        their reviewing permissions.
+        """
+        # Only execute this job if it's possible to send the emails.
+        if feconf.CAN_SEND_EMAILS and (
+                config_domain
+                .CONTRIBUTOR_DASHBOARD_REVIEWER_EMAILS_IS_ENABLED.value):
+            reviewer_ids = user_services.get_reviewer_user_ids_to_notify()
+            reviewers_suggestion_email_infos = (
+                suggestion_services
+                .get_suggestions_waiting_for_review_info_to_notify_reviewers(
+                    reviewer_ids))
+            email_manager.send_reviewers_contributor_dashboard_suggestions(
+                reviewer_ids, reviewers_suggestion_email_infos)
+
+
+class CronMailAdminContributorDashboardReviewIssuesHandler(
+        base.BaseHandler):
+    """Handler for mailing admins if there are issues that are causing a longer
+    reviewer turnaround time on the Contributor Dashboard.
+    """
+
+    @acl_decorators.can_perform_cron_tasks
+    def get(self):
+        """Sends each admin up to two emails: an email to alert the admins that
+        there are suggestion types that need more reviewers and/or an email
+        to alert the admins that specific suggestions have been waiting too long
+        to get reviewed.
+        """
+        # Only execute this job if it's possible to send the emails.
+        if feconf.CAN_SEND_EMAILS:
+            if config_domain.NOTIFY_ADMINS_REVIEWERS_NEEDED_IS_ENABLED.value:
```

Comment:
```suggestion
        if not (feconf.CAN_SEND_EMAILS or config_domain.NOTIFY_ADMINS_REVIEWERS_NEEDED_IS_ENABLED.value):
            return
```

Reply:
the double if's are there because there's another if coming in the next PR...there will be if config.domain.NOTIFY_ADMINS_SUGGESTIONS_WAITING... so I think I'm going to keep the format as is but thanks for the suggestion :)

Reply:
I don't have problems with double if, I'm trying to reduce the branching i.e, the nested if. (Like do the checks above and end the function in advance.)

Reply:
I think this is simpler:
if can_send_emails:
 if can_send_admin_reviewers_needed
 if can_send_admin_suggestions_waiting_too_long

than:
if (not can send_emails) or ((not can_send_admin_reviewers_needed) and (not can_send_admins_suggestions_waiting_too_long)
   return
if can_send_admin_reviewers_needed:
if can_send_admins_suggestions_waiting_too_long

Do you agree? It's only one nested if

Reply:
As per our discussion offline, I'll get rid of the nested if in the next PR to make the code better if more ifs get added here in the future. Also because Sandeep's suggestion makes the code more readable.

Reply:
The latter part has lots of duplicate checks.

I think you can write it this way:

```
if not can send_emails:
    return
if can_send_admin_reviewers_needed:
    ....
if can_send_admins_suggestions_waiting_too_long:
    ....
```

Reply:
> As per our discussion offline, I'll get rid of the nested if in the next PR to make the code better if more ifs get added here in the future. Also because Sandeep's suggestion makes the code more readable.

I'm fine with this! :)